other/dns-policy-and-dns-config/dns-policy-and-dns-config.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: dns-policy
```

Comment:
Can you update the name to be closer to the title annotation? Like maybe `change-dns-config-policy` or similar?